staging/src/k8s.io/apiserver/pkg/registry/rest/meta.go

```diff
@@ -25,15 +25,9 @@ import (
 )
 
 // FillObjectMetaSystemFields populates fields that are managed by the system on ObjectMeta.
-func FillObjectMetaSystemFields(ctx context.Context, meta metav1.Object) {
+func FillObjectMetaSystemFields(meta metav1.Object) {
 	meta.SetCreationTimestamp(metav1.Now())
-	// allows admission controllers to assign a UID earlier in the request processing
```

Comment:
@liggitt @deads2k do we use this in origin?

Reply:
don't seeing a call there at least to WithUID

Reply:
ok, https://github.com/kubernetes/kubernetes/issues/59775#issuecomment-389735867